### PR TITLE
chore: readonly not suppose to have default value

### PR DESCRIPTION
**Pull request by Ferror**
I wanted to use the stubs, and it seems that not all are defined correctly.

Btw. why not to move them to https://github.com/JetBrains/phpstorm-stubs?

### CODE REVIEW

File: ext/ddtrace.stub.php [hunk 1]
```diff
@@ -154,12 +154,12 @@ public function getLink(): SpanLink {}
         /**
          * @var SpanStack|null The parent stack, or 'null' if there is none
          */
-        public readonly SpanStack|null $parent = null;
```
Comment (bwoebi) [inline ext/ddtrace.stub.php:157]: Is there a reason you removed that readonly here?

**Comment by bwoebi** [comment]
@Ferror We need these stubs in our own repository as the arginfo.h are generated from these.

But good finds that they cannot have a default value.

File: ext/ddtrace.stub.php (hunk 1 above)
Comment (Ferror) [reply ext/ddtrace.stub.php:157]: Yes. You may notice that like [153](https://github.com/DataDog/dd-trace-php/pull/2180/files#diff-61e6285b55a2a83eebb2ae49717564d1a5f0307aee1d30d7df2bbf598826c5d1L153) defines whole class as a readonly, so it's invalid to duplicate it to property too.

**Comment by bwoebi** [comment]
I'll merge it next week, have to also regenerate stubs and test whether it then still works propertly.

**Review by bwoebi — APPROVED** [review]
SpanStack itself is not supposed to be readonly, that was a mistake, removed it there instead.

**Pull request merged**